flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -800,7 +800,7 @@ IntrinsicLibrary::genIntrinsicCall(llvm::StringRef name, mlir::Type resultType,
 
   auto runtimeCallGenerator = getRuntimeCallGenerator(name, soughtFuncType);
   return genElementalCall(runtimeCallGenerator, name, resultType, args,
-                          /* outline */ true);
+                          /* outline */ false);
```

Comment:
We should fix LLVM-IR dialect, etc.

Reply:
I agree, this is just a temporary workaround to allow Steve to run more tests. I did put a small FIXME to not forget setting it back to false.